.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "es5",
```

Comment:
A lot of these are already the default, like
"tabWidth": 2,
"trailingComma": "es5",
"jsxSingleQuote": false,
 "bracketSpacing": true,
 etc.
 
 I would not hardwire in the defaults again

Reply:
What would happen if a contributor has their `tabwidth` set to a different value? I thought we implement them here to keep consistency just in case some people have their settings altered?

Reply:
One of the reasons people specify the defaults for Prettier like this is that the project has been known to change the defaults in the past.  What then happens is you update the prettier dependency, and your entire tree gets reformatted for no good reason.

I think explicit defaults are useful, even if only to document our decisions.